exercises/security/README.md

```diff
@@ -16,6 +16,18 @@
 
 <details>
 <summary>What is a Certificate Authority?</summary><br><b>
+ 
+ A certificate Authority certifies the authenticity of the public key delivered by the website. It prevents [man-in-the-middle](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) attacks by providing a lot of information which identifie the public key. Importante information provided inside a [X.509](https://www.ssl.com/faqs/what-is-an-x-509-certificate/) certificate are like :
```

Comment:
Certificate Authority doesn't have to be related to websites. It's more generic than the specific implementation mentioned here.